### PR TITLE
[Scoper] Added namespaces to exclude

### LINKED ISSUE #29: Scoper
Symplify classes probably shouldn't be prefixed. I've got several errors like `Service of type RevealPrefix20220606\Symplify... not found. Did you add it to configuration file?`
But also `Service of type Reveal\LattePHPStanCompiler\LatteToPhpCompiler: Service of type RevealPrefix20220606\Latte\Parser required by $latteParser in LatteToPhpCompiler::__construct() not found. Did you add it to configuration file?`

All classes used as services should stay with original names, right?

I can try to prepare PR to scoper.php. It's about adding some exclude-namespaces isn't it?

### PR DESCRIPTION
**Pull request by lulco**
Resolves #29

### CODE REVIEW

File: scoper.php [hunk 1]
```diff
@@ -15,7 +15,17 @@
...
+        '#^PHPStan#',
+        '#^Reveal#',
+        '#^PhpParser#',
+        '#^Symplify\\\\Astral#',
+        '#^Symplify\\\\SmartFileSystem#',
+        '#^Symplify\\\\PackageBuilder#',
+    ],
+    'exclude-classes' => [
+        'Latte\Parser',
+    ],
```
Comment (TomasVotruba) [inline scoper.php:28]: This would actually break run on Latte 2 and 3. Those classes have different API, that's why we have to scope them.

Comment (lulco) [reply scoper.php:28]: hm, I understand... I was trying to solve this error:
```
Service of type Reveal\LattePHPStanCompiler\LatteToPhpCompiler: Service of type RevealPrefix20220711\Latte\Parser required by $latteParser in LatteToPhpCompiler::__construct() not found. Did you add it to configuration file?
```

Latte\Parser is registered as service in `packages/latte-phpstan-compiler/config/services.neon`
```neon
services:
    # latte
    - Latte\Parser
```

Maybe we could create RevealLatteParser class which will just extends Latte\Parser and register it as service?

Comment (TomasVotruba) [reply scoper.php:28]: I see. In that case it should be prefixed in the config as well. Probably with custom callback.

---

> Maybe we could create RevealLatteParser class which will just extends Latte\Parser and register it as service?

That's also option, but we should not bend the design just to make tool happy :). There is no application reason to extend it, unless we want to modify functionality.

Comment (lulco) [reply scoper.php:28]: Well unless somebody needs latte\parser as dependency in their services :) I hope there is no such case. 

Is there some example of such custom scoper callback?

Comment (lulco) [reply scoper.php]: ok I moved it to patchers, hope it is correct O:-) Is there some simple way how to test it? I don't want to reproduce all steps from github workflows

**Comment by lulco** [comment]
@TomasVotruba can you check this and merge if it is correct? I would like to use it in my projects :)

**Comment by TomasVotruba** [comment]
No prob, I fogot to handle this one.

The merge of these is usually the only way to test it, unless you run the scoper locally. Let's try it :)

**Pull request merged**